gateway-ha/pom.xml

```diff
@@ -15,10 +15,37 @@
     <name>gateway-ha</name>
 
     <properties>
+        <dropwizard.version>3.0.1</dropwizard.version>
```

Comment:
These properties could be kept in a single place - parent pom instead. It's simple to manage that way.